### ISSUE TITLE
throws error when running on Node for Windows

### ISSUE THREAD

**Issue by xer0x** [joyent/node-triton#83]
I tried running `triton` on windows. It doesn't work. This is what happens when typing triton in the 'Node.js command prompt' shortcut installed via https://nodejs.org/en/download/.

```
C:\Users\xer0x>triton
path.js:8
    throw new TypeError('Path must be a string. Received ' +
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:8:11)
    at Object.win32.resolve (path.js:130:5)
    at Object.<anonymous> (C:\Users\xer0x\AppData\Roaming\npm\node_modules\trito
n\lib\cli.js:37:23)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\xer0x\AppData\Roaming\npm\node_modules\trito
n\bin\triton:12:1)
```

**Comment by trentm** [comment]
First issue is usage of "$HOME" envvar.

Possibly could use: https://technet.microsoft.com/en-us/library/cc749104%28v=ws.10%29.aspx  windows envvars.... but this always turns into a cluster with windows.

**Comment by trentm** [comment]
published version 4.4.1
